server/events/policy_check_output_generator.go

```diff
@@ -0,0 +1,125 @@
+package events
+
+import (
+	"fmt"
+
+	"github.com/runatlantis/atlantis/server/events/command"
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/lyft/feature"
+)
+
+type PolicyCheckOutputKey struct {
+	ProjectName string
+	Workspace   string
+}
+
+type PolicyCheckOutputStore struct {
+	Store map[PolicyCheckOutputKey]*models.PolicyCheckSuccess
```

Comment:
because this is in memory, is the decision that we accept the risk of losing past policy results in a shutdown event?